src/components/Checkbox.js

```diff
@@ -18,6 +19,15 @@ const propTypes = {
     /** Should the input be disabled  */
     disabled: PropTypes.bool,
 
+    /** Children to wrap in AnimatedStep. */
```

Comment:
Does not seem correct. And it is not enough. What will happen if we don't pass it.

Reply:
Updated the comment. It has a default value of null. That we check inside the render function. May be we can move the default checkbox icon to a separate component and pass that as default prop?

Reply:
The current code is fine.